generators/entity/prompts.js

```diff
@@ -237,6 +238,45 @@ function askForRelationsToRemove() {
     }.bind(this));
 }
 
+function askForTableName() {
+    // don't prompt if there are no relationships
+    var entityTableName = this.entityTableName;
```

Comment:
L210 of index.js sets it back to undefined in the method _loadJson:
`this.entityTableName = this.fileData.entityTableName;`